test/parallel/utility-pallet.swapOperations.test.ts

```diff
@@ -19,12 +19,16 @@ import { Tokens } from "../../utils/tokens";
 import { getLiquidityAssetId } from "../../utils/tx";
 import { Staking } from "../../utils/Staking";
 import { Assets } from "../../utils/Assets";
+import { Market } from "../../utils/market";
 
 jest.spyOn(console, "log").mockImplementation(jest.fn());
 jest.setTimeout(1500000);
 
 let users: User[] = [];
 let tokenIds: BN[] = [];
+const poolIds: BN[] = [];
+let liqId: BN;
+let i: number = 0;
```

Comment:
declare this in a local scope, close to the while, & consider replace this by a for, looks simpler

Reply:
Moved